src/base/install/setupCOBRASystemPath.m

```diff
@@ -0,0 +1,31 @@
+function setupCOBRASystemPath()
```

Comment:
Wouldn't it be better to rename this function to `setupCOBRAbinaryPath()` or similar?

Reply:
probably best would be "addCOBRABinaryPathToSystemPath" (even though thats quite lengthy), but would actually say what it does.

Reply:
Would that be fine with you?

Reply:
Sounds good 👍